libs/wallet-ui/project.json

```diff
@@ -17,7 +17,8 @@
             "rm -rf ./dist/libs/wallet-ui",
             "nx run wallet-ui:build-css",
             "nx run wallet-ui:build-js",
-            "yarn tsc --emitDeclarationOnly --declaration --project libs/wallet-ui/tsconfig.lib.json"
+            "yarn tsc --emitDeclarationOnly --declaration --project libs/wallet-ui/tsconfig.lib.json",
```

Comment:
Let;'s create an issue to figure out the root cause of this, not sure if we want this going forward.

Reply:
THis is breaking new cypress tests, we should investigate why